content/millennium/dstu2.md

```diff
@@ -329,9 +329,29 @@ parameter:
 $ curl -i -H "Accept: application/json+fhir" "https://fhir-open.sandboxcerner.com/dstu2/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/MedicationOrder?patient=2744010&status=active"
 </pre>
 
-In this example, MedicationOrder is the FHIR<sup>®</sup> standard resource being accessed, while `patient` and `status` 
+In this example, MedicationOrder is the FHIR<sup>®</sup> standard resource being accessed, while `patient` and `status`
  are passed in the query string.
 
+ ### Resource Identity
+
+ Please note that no ids or identifiers in Millennium EHR are intended to be used outside of the context of their complete URL. A complete URL comprises of [service root url](http://hl7.org/fhir/dstu2/http.html#general), the resource, and parameters (if any).
+
+ For example, one must take into account the entire url and not simply the id or resource + id:
+
+     https://fhir-open.sandboxcerner.com/dstu2/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Person/1316024
+
+ In another context the id "Person/1316024" may identify another person entirely. In the following example a different resource may be returned because the context (service root url) has changed.
+
+     https://fhir-open.sandboxcerner.com/dstu2/d075cf8b-3261-481d-97e5-ba6c48d3b41f/Person/1316024
+
+ Similarly when considering an identifier one must consider its context even when normally it might be considered unique (social security number etc.):
```

Comment:
While social security number is an example of an identifier, one would hope that would be unique across multiple urls.  MRN might be a better example, as this will be organization-specific, rather than potentially existing across multiple systems.  So, rather than discuss uniqueness, you might explain that while some identifiers may exist across systems, this is not guaranteed, and (as above), could result in identifying different patients.  Therefore, they should be used in the context of the full url.